frontend/src/components/account/AccountPage.js

```diff
@@ -399,6 +402,32 @@ export default function AccountPage({
               ))}
             </Container>
           )}
+          {isOrganization && account.info.social_options.length > 0 && (
+            <>
+              <div className={classes.website}>
+                <Typography variant="caption"> {organizationTexts.find_us_here} </Typography>
```

Comment:
```suggestion
             
```
I feel like this is not really necessary

Reply:
Added different styling. Additionally, updated the logic so the text "find us here" only shows if there is a social link or website to display.